week-5/A-dom-manipulation/exercise.js

```diff
@@ -22,12 +54,57 @@ Task 3
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
+// var addTextBtn = document.querySelector("#addTextBtn");
+// addTextBtn.addEventListener("click", addText);
 
+////////////METHOD 2//////////
+divButtonChild[2].addEventListener("click", addText);
+
+function addText() {
+    var paragraph = document.createElement('p'); // here we're just creating it, element is not visible yet
+    addTextBtn.appendChild(paragraph);
+    paragraph.innerText = "Read me below.";
+    addTextBtn.removeEventListener("click", addText); // to stop at one paragraph
+    addTextBtn.style.backgroundColor = "Yellow"; //Just an extra
+}
```

Comment:
Good.  I like that you remove the event listener. 

If you read the spec carefully, it says the text should be added below the *buttons*, not below the text of one button!  The mechanism will be the same, anyway.